Header header
sesame_navigation/PointId[] points
